### DISCUSSION TITLE
yarn bootstrap is not working on Windows

### DISCUSSION THREAD

**Discussion by SoyDiego** [callstack/react-native-paper#4046] [Q&A]
Hi, I forked the repo and I tried to install the dependencies doing `yarn bootstrap` but nothing is happening.
In the beginining I thought maybe is a problem of my laptop because I was in my work. 
But I tried in 3 laptops more and always have the same problem.

I tried to reinstall yarn and verify the PATH but in the only place where is not working is inside the project.

In Linux works perfectly.

Here you have a gif:
![problem 2](https://github.com/callstack/react-native-paper/assets/19764334/5e48ff46-e16f-4876-b7bb-6ec18bf95a04)

Thanks!

**Comment by iM-GeeKy** [comment]
Hey @lukewalczak this question/issue, to no surprise, is surfacing on [react-native-paper-dates](https://github.com/web-ridge/react-native-paper-dates) which uses the same type of setup, and it made sense to reach out and pose the question since this repo has more traction and contributors than paper dates. Unfortunately, I don't have a windows machine, but was curious if you've ran across this as it could be deterring potential windows contributors.

**Comment by SoyDiego** [comment]
any help @lukewalczak?

Thanks
